MULTIPLE_STRIP_TESTING_WORKS_MIDI/MYHANDLE_NOE_ON.ino

```diff
@@ -2,54 +2,54 @@ void MyHandleNoteOn(byte channel, byte pitch, byte velocity) {
   brightness = map(velocity, 1 , 127 , 0 , 255);
   switch (pitch) {
     case 36: //bass
-    padHit = 5;
-    padName = "BASS";
+      padHit = 5;
+      padName = "BASS";
       bassled();

```

Comment:
I did make a comment about the padHit and padName, but now I realise you need them outside this scope. 
One thought though, you could remove all these calls to LedLights.ino and just call fill_solid directly. That will save a few cycles on the function call.

fill_solid(bassstrip, BASSLEDS, CHSV(bassColor, 255, brightness));